src/Symfony/Component/Debug/ErrorHandler.php

```diff
@@ -561,6 +561,8 @@ public static function handleFatalError(array $error = null)
 
         $handler = self::$reservedMemory = null;
         $handlers = array();
+        $previousHandler = null;
+        $sameHandlerLimit = 10;
```

Comment:
Why 10 and not 11? Or 9?

Reply:
I can update to 9 if you prefer :) any other number ?

Reply:
Hehe, You maybe had a thought about why you choose 10.  =)

Reply:
Not really :) 3 looked too low, 30 too high, 10 was in the middle :)